client/src/legacy/TinyMCE_sslink.js

```diff
@@ -131,14 +131,24 @@ jQuery.entwine('ss', ($) => {
       return Promise.resolve();
     },
 
-    buildAttributes(data) {
-      const anchor = data.Anchor && data.Anchor.length ? `#${data.Anchor}` : '';
-      const href = `${data.Link}${anchor}`;
+    /**
+     *
+     * @param {string|undefined} Anchor
```

Comment:
Looks like this docblock for deconstructed object may not be correct? https://jsdoc.app/tags-param.html#parameters-with-properties

Reply:
That's what PHPStorm auto-suggested. My guess is that's what it understands. I'm more concerned with keeping PHPStorm happy than JSDoc.